### DISCUSSION TITLE
How do I show the current frame?

### DISCUSSION THREAD

**Discussion by fconyt** [smplayer-dev/smplayer#115] [Q&A]
Apparently, in older versions of smplayer you could show the current frame of a video. I assume that's still there but the options menu seems to have been changed heavily since then. Where would I find this option?

**Comment by smplayer-dev** [comment]
If you select "mplayer" as multimedia engine in preferences then you can display the frame in the statusbar (menu Options -> Statusbar -> Frame counter. But depending on the video format this may not work.
